### ISSUE TITLE
Actually FNF on the Google Play Store

### ISSUE THREAD

**Issue by sonicyt1234** [luckydog7/Funkin-android#937]
So I was watching YouTube yesterday, And I found an ad of a game called "FNF Mod" And it had Selever on the icon, And it looked legit, I pressed install, It installed, And then... I played the Google Play Store version, And it was actually a port of the Kade Engine version of FNF, Trust me, It was legit, This is what the icon looks like:

![20210523_084029](https://user-images.githubusercontent.com/81123154/119259014-955e1880-bba2-11eb-8e9a-dd54067bbd51.jpg)

**Comment by georgechen1236** [comment]
Poggees

**Comment by sonicyt1234** [comment]
No, For real,  https://play.google.com/store/apps/details?id=com.fmf.mod

**Comment by HotRatDLol** [comment]
Thx dude

**Comment by sonicyt1234** [comment]
Amazing right? BECAUSE THERE ARE NO BLACK SQUARES, AND THERE ARE LESS CRASHES!!!!

**Comment by Flamehere** [comment]
Yeah it have a superior port

**Comment by Flamehere** [comment]
But play it is kinda cheating because even if you press wrong note you dont lose score
